lib/IDE/SyntaxModel.cpp

```diff
@@ -418,6 +418,21 @@ CharSourceRange parameterNameRangeOfCallArg(const TupleExpr *TE,
   return CharSourceRange();
 }
 
+static CharSourceRange findDocCommentRange(DeclAttributes Attrs) {
+  for (auto Attr : Attrs) {
+    if (auto DocAttr = dyn_cast_or_null<RawDocCommentAttr>(Attr)) {
+      return DocAttr->getCommentRange();
```

Comment:
> Is that desired behavior?

Not really 😅 

I've tested it and it indeed returns everything. Do you have any suggestions on how to improve this?